client/package.json

```diff
@@ -20,13 +20,21 @@
     "remark-html": "^15.0.1"
   },
   "devDependencies": {
-    "@types/jest": "^29.4.0",
+    "@babel/core": "^7.21.3",
```

Comment:
if those libs are necessary only for jest, then you can add it in dev dependencies

Reply:
what do you mean? but they are already in devDependencies

Reply:
Sorry,  that's true